docs/iot/intro.md

```diff
@@ -12,6 +12,10 @@ ms.prod: dotnet
 
 .NET runs on various platforms and architectures. Common Internet of things (IoT) boards, such as Raspberry Pi and Hummingboard, are supported. IoT apps typically interact with specialized hardware, such as sensors, analog-to-digital converters, and LCD devices. The .NET IoT Libraries enable these scenarios.
 
+## Video introduction
```

Comment:
Is this header required? If this is an introduction you want people to read, I would add an intro paragraph to this heading that gives a quick overview about what they'll learn if they watch the video. Personally, I hate videos and usually just skip past them. So intro information about it would help me decide if it's worth my time.

Or, should the header be removed and the video just sit with the intro paragraph? That's the template you've used with the other articles in the PR.